buildengine/build.go

```diff
@@ -45,15 +45,16 @@ func buildModule(ctx context.Context, sch *schema.Schema, module Module) error {
 	}
 
 	if err != nil {
+		errs := errors.UnwrapAllExcludingIntermediaries(err)
```

Comment:
added `UnwrapAllExcludingIntermediaries` because we're basically decomposing this error and then rejoining it (L:59 `errors.Join(errs...)`), so the presence of the intermediate errors creates a bunch of duplication/spam